[Click to Call] Updates to improve clarity on README for low code approach to get going. (#253)
Azure-Samples/communication-services-javascript-quickstarts: ui-library-click-to-call/README.md

### Setting up the bundle

> In the `index.html` file find the `userId`, `token`, and `displayName` values and enter your information like is used in the [React app](#swap-placeholders-for-identifiers).

Reviewer: ```suggestion
In the `index.html` file find the `userId`, `token`, and `displayName` values and enter your information, as similarly done in the [React app](#swap-placeholders-for-identifiers).
```

### Running the bundle

> There is no need to install any packages, using [npm](https://nodejs.org/en/learn/getting-started/an-introduction-to-the-npm-package-manager) from the root of the `ui-library-click-to-call` folder run 

Reviewer: ```suggestion
You do not need need to install any packages.
Rather, using  [npm](https://nodejs.org/en/learn/getting-started/an-introduction-to-the-npm-package-manager) from the root of the `ui-library-click-to-call` folder run 
```

```bash
npm run start:js
```

>  after in the browser go to [http://localhost:3000](http://localhost:3000) you will then be greeted with a simplified example of click to call with the `CallComposite` configuration page as the start point after the `show calling experience` button is actioned.

Reviewer: ```suggestion
now in the browser, go to [http://localhost:3000](http://localhost:3000) you will then be greeted with a simplified example of click to call with the `CallComposite` configuration page as the start point after the `show calling experience` button is actioned.
```
